manual-of-style/index.html

```diff
@@ -910,7 +913,7 @@ <h3>Using Examples</h3>
 <li>
 <p>Use <abbr title="Cascading Style Sheets">CSS</abbr> with
 <code>div</code> elements to mark up examples, as is done in the
-<a href="https://www.w3.org/TR/html51/document-metadata.html#example-2ced58ee"
+<a href="https://html.spec.whatwg.org/multipage/semantics.html#document-metadata"
 title="Section 4.2. Document metadata">HTML 5.1</a>
```

Comment:
```suggestion
title="Section 4.2. Document metadata">HTML Living Standard</a>
```
Update link text correspondingly with update to link href